cmd/admin.go

```diff
@@ -59,6 +59,11 @@ var (
 				Value: "custom/conf/app.ini",
 				Usage: "Custom configuration file path",
 			},
+			cli.StringFlag{
```

Comment:
It should be `cli.BoolFlag`. Also why default to false if from UI default is true?